### PR TITLE
Added basic dependencies

### PR DESCRIPTION
**Pull request by dembasiby**
Added Spring Data JPA, MySQL adapter, Lombok, and Flyway dependencies, and designed first version of API contracts

### CODE REVIEW

File: user-management-service/pom.xml [hunk 1]
```diff
@@ -22,6 +22,29 @@
...
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.mysql</groupId>
+            <artifactId>mysql-connector-j</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.flywaydb/flyway-mysql -->
+        <dependency>
```
Comment (ajeetcloud) [inline user-management-service/pom.xml]: pls remove unrequired dependencies

**Review by ajeetcloud — CHANGES_REQUESTED** [review]
remove commented lines in pom.xml

File: user-management-service/pom.xml (hunk 1 above)
Comment (dembasiby) [reply user-management-service/pom.xml]: Done

**Comment by dembasiby** [comment]
Pom file cleaned

**Pull request closed without merging**